react-frontend/src/core-components/label/LabelComponent.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Form } from "react-bootstrap";
+import PropTypes from "prop-types";
```

Comment:
For "react-bootstrap" component this is not required. Please remove this line and check